### ISSUE TITLE
[Feature request]The "Allowed IPs" of "Wireguard Client" may be set globally

### ISSUE THREAD

**Issue by jearton** [ngoduykhanh/wireguard-ui#99]

**Comment by MANKUD** [comment]
Yes, this would be really nice rather than manually configuring for each client.

**Comment by zilexa** [comment]
Yes please!

**Comment by Azraeht** [comment]
This could be a very great feature. We forked your project to add in our organization : https://github.com/CybelAngel/wireguard-ui/commit/57d4a41094747b036ae440c340d00355d49ed47c

It could be really usefull for us if this feature could be integrated.

**Comment by zilexa** [comment]
Could this be implemented easily since the code is already available?
